testing/commontest.py

```diff
@@ -13,9 +13,9 @@
 RBBin = os.fsencode(shutil.which("rdiff-backup"))
 
 # Working directory is defined by Tox, venv or the current build directory
-abs_work_dir = os.getenvb(
-    b'TOX_ENV_DIR',
-    os.getenvb(b'VIRTUAL_ENV', os.path.join(os.getcwdb(), b'build')))
+abs_work_dir = os.getenv(
+    'TOX_ENV_DIR',
+    os.getenv('VIRTUAL_ENV', os.path.join(os.getcwdb(), b'build'))).encode()
```

Comment:
Two things here:
1. I think you need to use `getcwd` and `'build`' (both without b) or you might get a mix of strings and bytes (it happens only if you use the test outside of tox, but I do this a lot while developing).
2. instead of str.encode for paths and similar, please use [os.fsencode](https://docs.python.org/3/library/os.html#os.fsencode) to be consistent with the rest of the code.